app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -198,11 +199,11 @@ public View getView(int position, View convertView, ViewGroup parent) {
         if (viewType == VIEW_TYPE_NAV) {
             v = getNavView((String) getItem(position), position, convertView, parent);
         } else if (viewType == VIEW_TYPE_SECTION_DIVIDER) {
-            v = getSectionDividerView(convertView, parent);
+            v = getSectionDividerView(parent);
         } else {
             v = getFeedView(position, convertView, parent);
         }
-        if (v != null && viewType != VIEW_TYPE_SECTION_DIVIDER) {
+        if (v != null && (viewType != VIEW_TYPE_SECTION_DIVIDER || UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE)) {
```

Comment:
Why is this needed? I do not think that the section divider should stay selected after clicking.

Reply:
As I said, I could not get this working, so I added some ways and maybe useless stuff :sweat_smile: 
I reverted the commit and made a new clear version